src/test/java/api/requests/RequestsAPIRetrievalTests.java

```diff
@@ -869,4 +868,57 @@ private void assertThatRequestsHavePatronComments(MultipleJsonRecords requests)
       hasJsonPath("patronComments", "Comment 5")
     ));
   }
+
+
+  @Test
+  void printDetailsTest() {
+    UserResource printDetailsRequester = usersFixture.charlotte();
+    UUID printDetailsRequesterId = printDetailsRequester.getId();
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+
+    final IndividualResource workAddressType = addressTypesFixture.work();
+
+    final IndividualResource charlotte = usersFixture.charlotte(
+      builder -> builder.withAddress(
+        new Address(workAddressType.getId(),
+          "Fake first address line",
+          "Fake second address line",
+          "Fake city",
+          "Fake region",
+          "Fake postal code",
+          "Fake country code")));
+
+
```

Comment:
Do we need two blank lines here?

Reply:
fixed: https://github.com/folio-org/mod-circulation/pull/1492/commits/597143297af4b5344f832f462d339b4dbb58dd3c